gprofiler/main.py

```diff
@@ -27,7 +27,15 @@
 from .java import JavaProfiler
 from .perf import SystemProfiler
 from .python import get_python_profiler
-from .utils import is_root, run_process, get_iso8061_format_time, resource_path, log_system_info, TEMPORARY_STORAGE_PATH
+from .utils import (
```

Comment:
@Jongy wtf is this formatting?

Reply:
Default `black` settings?

Reply:
lol looks ugly (but effective git diff wise, since each is at its own line)

Reply:
I don't really mind how it looks tbh, as long as it just works (and yeah, the git diff is nice)